go/ethclient/mgmtcontractlib/mgmt_contract_ABI.go

```diff
@@ -1,23 +1,4 @@
-package contracts
-
-import (
-	"strings"
-
-	"github.com/ethereum/go-ethereum/accounts/abi"
-)
-
-// no point starting the node with invalid contracts
-func init() { //nolint:gochecknoinits
-	var err error
-	MgmtContractABIJSON, err = abi.JSON(strings.NewReader(MgmtContractABI))
-	if err != nil {
-		panic(err)
-	}
-}
-
-// MgmtContractABIJSON holds the compiled ABI
-// TODO it would be nice to have a auto generation script here
-var MgmtContractABIJSON = abi.ABI{}
+package mgmtcontractlib
 
 const (
 	AddRollupMethod     = "AddRollup"
```

Comment:
DepositMethod is not needed

Reply:
Removed, updated the contract and regenerated the abi / bytecode